File: 4-go/dtos/product.go
package dtos

type Product struct {
	ID          int64   `json:"id"`
	Name        string  `json:"name"`
	Category    int     `json:"category"`
	Price       float32 `json:"price"`
	CreatedDate string  `json:"createdDate"`
	Description string  `json:"description:`
	Available   int     `json:"available"`
}

File: 5-react/my-app/src/models/product.js
export class Product {
    id;
    name;
    category;
    price;
    createdDate;
    description;
    available;

    constructor(
        id,
        name,
        category,
        price,
        created_date,
        createdDate,
        available
    ) {
        this.id = id;
        this.name = name;
        this.category = category;
        this.price = price;
        this.createdDate = created_date;
        this.description = createdDate;
        this.available = available;
    }
}